File: 0009-palindrome-number/0009-palindrome-number.py
class Solution(object):
    def isPalindrome(self, x):
        """
        :type x: int
        :rtype: bool
        """
        string = list(str(x))
        print(string)
        original = list(str(x))
        string.reverse()
        if string == original:
            return True
        return False
        

File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String value = String.valueOf(x);
        System.out.println(value);
        String copy = new String();
        for(int i=0; i < value.length(); i++){
            copy = value.charAt(i) + copy;
        }
        
        System.out.println(copy);
        
        if(value.equals(copy))
        {return true;}
        return false;
    }
}